CMakeLists.txt

```diff
@@ -14,19 +14,12 @@ include(FindOpenCL)
 
 include(FindComputeCpp)
 
-include(FindBLAS)
+message(STATUS "Searching for BLAS in ${BLAS_DIR}")
 
-if (DEFINED OPENBLAS_ROOT)
-  set(USE_OPENBLAS ON)
-else()
-  set(USE_OPENBLAS OFF)
-endif()
+# Hackily set this to see if it fixes it! 
+set(BLAS_DIR /tmp/OpenBLAS/build)
```

Comment:
Is this for travis?

Reply:
This is for travis - I was basically just trying to debug *why* it wasn't finding OpenBLAS, before I realised it was a CMake version issue